### DISCUSSION TITLE
Dividing telemetry sensor values.

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#5089] [Q&A] [answered]
Hi everyone! I own many different RC models, and they all have different cells and battery types (LiPo/LiHV). So instead of trying to memorize what’s dead, storage, and full for 2s, 3s, 4s, and 6s, I just memorized it per cell (about 3.2 is about dead, 3.8 is storage, 4.2 or 4.3 for LiHV is full).

I want the approximate cell voltage displayed on my telemetry screen on my RadioMaster MT12, but I don’t know how to do it reliably. (I have a battery sensor which is accurate, and I have two instances so I can have the cell voltage and also have the standard reading.) 

what settings do I change in the sensor settings to divide it by 2, 3, 4, and 6? (To get the average cell voltage)

I looked at the manual but I’m still not **quite** sure what to do.

thanks for any help!

![IMG_3079](https://github.com/EdgeTX/edgetx/assets/142270525/0d9f534e-b7b4-45cd-93d1-fd503bcc9484)

**Comment by lightbulbjim** [comment, marked as answer]
Use the ratio parameter. 25.5 is equivalent to 1 (8 bit int/10).

So for example if you have a 3s battery you want 25.5/3 = 8.5 for the ratio value.

Once you have the ratio set correctly it's usually a good idea to put a trusted voltmeter on the battery and tweak the offset parameter to account for any inaccuracies in the RX sensor.

**Reply by inventor7777** [reply]
Exactly what I needed; thanks!
